tests/ssl-opt.sh

```diff
@@ -2443,7 +2443,7 @@ requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
 requires_config_enabled MBEDTLS_DEBUG_C
 requires_config_enabled MBEDTLS_SSL_CLI_C
 requires_config_enabled MBEDTLS_SSL_SRV_C
-requires_config_enabled MBEDTLS_ECDSA_C
+requires_config_enabled MBEDTLS_KEY_EXCHANGE_ECDH_ECDSA_ENABLED
```

Comment:
I don't think this one or the next one are correct. This should probably use the `requires_pk_alg "ECDSA"`  that I was suggesting previously.